app/models/tweet.rb

```diff
@@ -3,4 +3,13 @@ class Tweet < ActiveRecord::Base
 
```

Comment:
has_many :user_like_tweets

then, you can do something like 

 - user_like_tweets.create user_id: user.id unless user_like_tweets.exists? user_id: user.id
- user_like_tweets.find_by user_id: user.id

or 

user_like_tweets << aUserLikeTweet